lib/query/query_donations.rb

```diff
@@ -42,7 +43,21 @@ def self.campaign_export(campaign_id)
               .from('users')
               .add_join('profiles', 'profiles.user_id = users.id')
               .as("users").parse, "users.profiles_id=campaigns.profile_id")
-     .group_by("donations.id", "supporters.id", "payments.id", "payments.gross_amount", "users.email")
+     .group_by(
+        "donations.id",
+        "supporters.id",
+        "payments.id",
+        "payments.gross_amount",
+        "users.email",
+        "supporters.id",
+        "supporter_addresses.id",
+        "supporter_addresses.supporter_id",
+        "supporter_addresses.address",
+        "supporter_addresses.state_code",
+        "supporter_addresses.city",
+        "supporter_addresses.zip_code",
+        "supporter_addresses.country"
+      )
```

Comment:
I'm not sure I understand this change. What's the purpose?

Reply:
I was having a `column "supporter_addresses.x" must appear in the GROUP BY clause or be used in an aggregate function` error. I may be wrong, but I understood it was because of the outcome from the left join with `supporter_addresses` because a supporter can have multiple `supporter_addresses` - of course that if we are joining by `primary_address_id`, we'll have only one result, but the query doesn't know that.

Reply:
Is there any other alternative for this that you can think of?